linkis-computation-governance/linkis-engineconn-manager/linkis-engineconn-manager-core/src/main/scala/org/apache/linkis/ecm/core/launch/ProcessEngineConnLaunch.scala

```diff
@@ -99,6 +101,32 @@ trait ProcessEngineConnLaunch extends EngineConnLaunch with Logging {
     Utils.tryFinally(socket.getLocalPort)(IOUtils.closeQuietly(socket))
   }
 
+  private def findAvailPortByRange(): Int = {
+    val separator = ":"
```

Comment:
Maybe it's more accurate to use `-` as a separator

Reply:
The new method `findAvailPortByRange()` needs to add the corresponding unit test code